build-system/tasks/presubmit-checks.js

```diff
@@ -836,6 +836,12 @@ const forbiddenTermsSrcInclusive = {
       'src/service/url-replacements-impl.js',
     ],
   },
+  '\\.setNonBoolean\\(': {
```

Comment:
I actually think all storage usage should be reviewed, including set boolean.

We might introduce a rule to check `storageForDoc`

Reply:
we have the check already.